slam_gmapping/include/slam_gmapping/slam_gmapping.h

```diff
@@ -100,16 +107,18 @@ class SlamGmapping : public rclcpp::Node{
     std::shared_ptr<std::thread> transform_thread_;
 
     std::string base_frame_;
-    std::string laser_frame_;
```

Comment:
This line has been moved below. The change does not make any difference. We can keep the original declaration

Reply:
OK. I have reverted this to the original code.